src/QMCHamiltonians/tests/test_SOECPotential.cpp

```diff
@@ -56,9 +56,10 @@ class TestSOECPotential
                               const RefVectorWithLeader<TrialWaveFunction>& twf_list,
                               const RefVectorWithLeader<ParticleSet>& p_list,
                               const std::optional<ListenerOption<Real>> listener_opt,
-                              bool keep_grid)
+                              bool keep_grid,
+                              bool exact_spin)
```

Comment:
Apart from accordingly removing `exact_spin`, it is better to remove the first argument `SOECPotential& so_ecp`. It is use can be replaced with the leader of `o_list`.

Reply:
should be fixed